sdk/storage/azure-storage-datamovement/src/tasks/download_blob_directory_task.cpp

```diff
@@ -0,0 +1,34 @@
+#include "azure/storage/datamovement/tasks/download_blob_directory_task.hpp"
+#include "azure/storage/datamovement/tasks/download_blob_to_file_task.hpp"
+#include "azure/storage/datamovement/scheduler.hpp"
+namespace Azure { namespace Storage { namespace Blobs { namespace _detail {
+  void DownloadBlobDirectoryTask::Execute() 
```

Comment:
There's still some distance away from completion for this one.

We need to do paging here. That's important because there could be a huge amount of blobs and it would overwhelm memory.

How do you like this: we remove this directory-downloading functionality in this PR, and we'll add this back with better completeness later. Or do you want to finish it before merging?

Reply:
Yes, I'm thinking we may need to change scheduler for it. I can remove the function, and leave an empty framework for it.